Add blog post on making a cluster SCS-compliant (#917)
SovereignCloudStack/website: _i18n/en/_posts/blog/2024-05-13-cost-of-making-an-openstack-cluster-scs-compliant.md

```bash
openstack flavor set --property aggregate_instance_extra_specs:ssd=true SCS-2V-4-20s
openstack flavor set --property aggregate_instance_extra_specs:ssd=true SCS-4V-16-100s
```

Without this additional configuration, workload might be scheduled to non-SSD-capable hosts.

Lastly, the entropy standard doesn't require any work as long as you only provide VM images with Linux kernels version 5.18 and newer.
This is the case for all the standard images mentioned in *SCS-0104-v1* (which we have already configured above), and as such,
VMs spawned from them will have enough entropy.
Otherwise, you'll need to ensure sufficient entropy is available by some other means, e.g., with special CPU instructions such as
`RDSEED`/`RDRAND`.
The details are out of scope for this blog post, but the [*SCS-0101-v1* standard](https://docs.scs.community/standards/scs-0101-v1-entropy/)
lists some possible implementation approaches as a starting point.

## But at what cost?

A relevant question we had with this was the cost of adopting all these standards for an OpenStack cluster.
Based on the work time after the cluster setup and without debugging, since this part would be minimized on a second

> or third attempt at this, we estimated around 4-6 man-hours for a minimal, freshly installed cluster. Doing this multiple

Reviewer: ```suggestion
r third attempt at this, we estimated around 4-6 person-hours for a minimal, freshly installed cluster. Doing this multiple
```

> times could reduce this time even further so something like 1-3 man-hours.

Reviewer: ```suggestion
times could reduce this time even further so something like 1-3 person-hours.
```

Now it is important to mention, that we had nearly ideal circumstances, since there was neither hardware missing nor
the additional costs associated with adopting an older cluster which already contained data.
If an older cluster needs to be adopted to the standards, it would be necessary to add metadata to existing images, possibly

> change their names and (if desired) change flavor names to the SCS naming schema. This would require significantly more time

Reviewer: ```suggestion
change their names and (if desired) change flavor names to the SCS naming schema or better add the SCS- flavors as additional flavors to avoid breaking users of the previously existing flavors. This would require significantly more time
```

> to do; we estimate this with around 0.2-1 man-hours per image or flavor. If this needed to be done more often or multiple

Reviewer: ```suggestion
to do; we estimate this with around 0.2-1 person-hours per image or flavor. If this needed to be done more often or multiple
```

> times, some form of automation would be recommended, but this would also incur some upfront man-hour cost.

Reviewer 2: ```suggestion
times, some form of automation would be recommended, but this would also incur some upfront person-hour cost.
```

Additional costs could come up if no SSDs were provided for the cluster. This would require a hardware upgrade, incurring

> cost for hardware (120-200€ per Terabyte), server downtime as well as man-hours. The actual costs here are hard to estimate

Reviewer 2: ```suggestion
cost for hardware (120-200€ per Terabyte at the time of writing this blog post), server downtime as well as man-hours. The actual costs here are hard to estimate
```

and would probably change from case to case.

> Nonetheless, it is to mention that in most cases, SCS compliance should be easily achievable for most OpenStack clusters

Reviewer 2: ```suggestion
Nonetheless, it is worth to mention that in most cases, SCS compliance should be easily achievable for most OpenStack clusters
```

without having too much overhead in adoption costs. This could obviously change in the future with the arrival of

> additional standards.

Reviewer: ```suggestion
additional standards. Also, OpenStack setups that diverge from the vanilla upstream configurations significantly may cause additional effort to get into compliance.
```

  </a>
</figure>

> At the time of writing, a vanilla Yaook deployment is not SCS-compliant and, as such, it is the ideal playground for our evaluation.

Reviewer 3: I think it is `SCS compliant` when it stands alone. It is `SCS-compliant` when in front of a noun.
